for_other_cantons.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan  3 09:20:09 2022
+
+@author: eduardoaraujo
+"""
+import pandas as pd 
+import numpy as np
+import streamlit as st
+import plotly.graph_objects as go
+from forecast import download_button
+from get_data import get_canton_data  
+from sqlalchemy import create_engine
+engine = create_engine("postgresql://epigraph:epigraph@localhost:5432/epigraphhub")
+
+
+
+#@st.cache
+def get_curve(curve):
+    '''
+    Function to get the cases and hosp data in the database
+    '''
+    df = pd.read_sql_table(f'foph_{curve}', engine, schema = 'switzerland', index_col = 'datum', columns=[ 'geoRegion', 'entries'])
+    df.index = pd.to_datetime(df.index)
+    
+    return df
+
+def plot_cases(canton):
+    
+    ''''
+    Function to plot the new cases according to FOPH in any canton
+    
+    params canton: canton to plot the data
+    
+    return[0] plotly figure
+    return[1] last data of new cases reported
+    
+    '''
+    
+    df = get_curve('cases')
+    
+    df = df.loc[df.geoRegion == canton]
+    df.sort_index(inplace = True)
+    df = df['2021-08-01':]
+    
+    # computing the rolling average 
+    m_movel = df.rolling(7).mean().dropna()
+    
+    fig = go.Figure()
+        
+    title = f"New Cases - {canton}"
+    
+    fig.update_layout(width=900, height=500, title={
+                'text': title,
```

Comment:
@eduardocorrearaujo we could leave only canton name in the title of figures, the description of the data is already in the `yaxis_title`